cli-tests/tests/pmm-cli/server/upgrade.spec.ts

```diff
@@ -1,17 +1,17 @@
-import { test, expect } from '@playwright/test'
-import * as cli from '@helpers/cliHelper'
-import PMMRestClient from '@tests/support/types/request'
-import { teardown } from '@tests/helpers/containers'
+import { test, expect } from '@playwright/test';
+import * as cli from '@helpers/cliHelper';
+import PMMRestClient from '@tests/support/types/request';
+import { teardown } from '@tests/helpers/containers';
 
-test.describe.configure({ mode: 'parallel' })
+test.describe.configure({ mode: 'parallel' });
 
-test.describe('Upgrade PMM Server', async () => {
+test.describe('pmm-bin: server upgrade tests', async () => {
   test('shall respect relevant flags', async () => {
-    const oldContainerName = 'pmm-server-upgrade-1'
-    const newContainerName = 'pmm-server-upgrade-1-new'
-    const oldImage = 'percona/pmm-server:2.32.0'
-    const newImage = 'percona/pmm-server:2.33.0'
-    const volumeName = 'pmm-data-upgrade-1'
+    const oldContainerName = 'pmm-server-upgrade-1';
+    const newContainerName = 'pmm-server-upgrade-1-new';
+    const oldImage = 'percona/pmm-server:2.32.0';
+    const newImage = 'percona/pmm-server:2.33.0';
+    const volumeName = 'pmm-data-upgrade-1';
```

Comment:
There's a lot of these "styling" updates.
I recommend using a library to check and apply styles, such as `Standard.js`